Code/GraphMol/MMPA/MMPA.cpp

```diff
@@ -394,8 +408,33 @@ bool fragmentMol(const ROMol& mol,
 
   // loop to generate every cut in the molecule
   BondVector_t bonds_selected;
-  processCuts(0, maxCuts, bonds_selected, matching_bonds, mol, res);
+  processCuts(0, minCuts, maxCuts, bonds_selected, matching_bonds, mol, res);
+  return true;
+}
+
+bool fragmentMol(const ROMol& mol,
+                 std::vector<std::pair<ROMOL_SPTR, ROMOL_SPTR> >& res,
+                 const std::vector<unsigned int>& bondsToCut,
+                 unsigned int minCuts,
+                 unsigned int maxCuts) {
+  std::vector<BondVector_t> matching_bonds;  // List of matched query's bonds
+
+  BOOST_FOREACH(unsigned int i, bondsToCut) {
+    const Bond *bond = mol.getBondWithIdx(i);
+    if(bond) {
```

Comment:
This test is unnecessary: `getBondWithIdx()` will either return a result or generate an exception (there's a POSTCONDITION that the result is non-null).